src/goto-symex/goto_symex_state.cpp

```diff
@@ -27,6 +27,14 @@ Author: Daniel Kroening, kroening@kroening.com
 
 static void get_l1_name(exprt &expr);
 
+static void set_l2_indices(
+  symex_level0t level0,
```

Comment:
Commented on the other PR, but can you explain the by-value passing here?

Reply:
the reference was omitted by mistake, but this reveals that this level0 is actually constant, I going to replace the symex_level0t by a function